mpas_analysis/polar_regions.cfg

```diff
@@ -425,6 +460,55 @@ makeTables = True
 iceShelvesInTable = ['all']
 
 
+[timeSeriesOceanRegions]
+## options related to plotting time series of groups of ocean regions
+
+# the names of region groups to plot, each with its own section below
+regionGroups = ['Arctic Ocean Regions']
+
+
+[timeSeriesArcticOceanRegions]
+## options related to plotting time series of Arctic Ocean regions
```

Comment:
I think it makes sense for this section to go into `default.cfg` instead of here.

Reply:
Resolved by https://github.com/MPAS-Dev/MPAS-Analysis/pull/896/commits/4df40486175cd6b95effabf899062eaafe197669